### PR TITLE
Update cast-and-convert-transact-sql.md

### PR DESCRIPTION
**Pull request by oscillations**
declare @val binary(16) = 0xFF

select CONVERT(nvarchar(50), @val, 1) [1], CONVERT(nvarchar(50), @val, 2) [2]

### CODE REVIEW

**Comment by PRMerger19** [comment]
@oscillations : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@oscillations : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by oscillations** [comment]
Is there something else you need to merge & fix the documentation?

File: docs/t-sql/functions/cast-and-convert-transact-sql.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ For a **binary(n)**, **char(n)**, **varbinary(n)**, or **varchar(n)** *expressio
...
... a character expression. The *expression* must have an **even** number of hexadecimal digits (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, A, B, C, D, E, F, a, b, c, d, e, f). If the *style* is set to 1, the expression must have 0x as the first two characters. If the expression contains an odd number of characters, or if any of the characters is invalid, an error is raised.<br /><br /> If the length of the converted expression exceeds the length of the *data_type*, the result is right truncated.<br /><br /> Fixed length *data_type*s larger than the converted result have zeros added to the right of the result.<br /><br /> A *data_type* of type character requires a binary expression. Each binary character is converted into two hexadecimal characters. If the length of the converted expression exceeds the length of the *data_type*, it will be right truncated.<br /><br /> For a fixed size character type *data_type*, if the length of the converted result is less than its length of the *data_type*, spaces are added to the right of the converted expression, to maintain an even number of hexadecimal digits.<br /><br /> The characters 0x will NOT be added to the left of the converted result for *style* 2.|
```
Comment (markingmyname) [inline docs/t-sql/functions/cast-and-convert-transact-sql.md:164]: Remove all instances of the word "will."

We do not use future text with tech writing.

Thank you for your contribution.

Once you edit that, I can approve it.

```suggestion
|**1**, **2**|For a binary *data_type*, the expression must be a character expression. The *expression* must have an **even** number of hexadecimal digits (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, A, B, C, D, E, F, a, b, c, d, e, f). If the *style* is set to 1, the expression must have 0x as the first two characters. If the expression contains an odd number of characters, or if any of the characters is invalid, an error is raised.<br /><br /> If the length of the converted expression exceeds the length of the *data_type*, the result is right truncated.<br /><br /> Fixed length *data_type*s larger than the converted result has zeros added to the right of the result.<br /><br /> A *data_type* of type character requires a binary expression. Each binary character is converted into two hexadecimal characters. Suppose the length of the converted expression exceeds the length of the *data_type*. In that case, it's truncated.<br /><br /> For a fixed size character type *data_type*, if the length of the converted result is less than its length of the *data_type*, spaces are added to the right of the converted expression to maintain an even number of hexadecimal digits.<br /><br /> The characters 0x are not added to the left of the converted result for *style* 2.|  
```
Once you edit that, I can approve it.

**Comment by PRMerger14** [comment]
@markingmyname : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by markingmyname** [comment]
#sign-off

**Pull request merged**